pkg/authorization/middleware.go

```diff
@@ -59,6 +60,11 @@ func (m *Middleware) WrapHandler(handler func(ctx context.Context, w http.Respon
 
 		if err := authCtx.AuthZRequest(w, r); err != nil {
 			logrus.Errorf("AuthZRequest for %s %s returned error: %s", r.Method, r.RequestURI, err)
+			if strings.Contains(err.Error(), ErrInvalidPlugin.Error()) {
```

Comment:
@anusha-ragunathan I don't understand this block. What are we doing with `m.plugins = authCtx.plugins` and why are we doing it if the error contains ErrInvalidPlugin ?